src/components/ProgressBar/ConfigurableDoubleProgressBar.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+	<div class="wmde-banner-double-progress" :style="{
+		'--wmde-banner-progress-bar-width': progressBarContent.percentageTowardsTarget + '%',
+		'--wmde-banner-progress-bar-time-width': currentCampaignTimePercentage + '%'
+	}">
+		<div class="wmde-banner-double-progress-labels" v-if="withExtraLabels">
```

Comment:
You could omit the `withExtraLabels` and [check for the existence of one of the label slots](https://vuejs.org/api/component-instance.html#slots) with `$slots['extra-label-left'] || $slots['extra-label-right']` instead.